addons/mail/models/mail_link_preview.py

```diff
@@ -107,6 +107,8 @@ def _get_link_preview_from_url(self, url, request_session):
             'image/tiff',
             'image/x-icon',
         )
+        if not response.headers.get('Content-Type'):
+            return False
```

Comment:
Would it be fair to still attempt to retrieve opengraph data by assuming the content type was html? Could just call the method anyway and return False if it fails.

Reply:
I don't think we can try to do that as _get_link_preview_from_html will throw an error in some cases (for example if the link is a mp4 video).

Reply:
Yes, but I was wondering if it might make sense to still attempt it when there's no content type, in a try block, and just return False on parsing exceptions.

Basically if in this case we expected to have opengraph data, and the only issue was that their content type was missing for some reason, might have made sense. But it doesn't seem to be the case, and it feels like if you don't send a content-type you're unlikely to have opengraph data on your page.

So seems fine to keep as is :)